cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -314,6 +316,57 @@ def trigger(self,component_name,service_name,blocking=True):
 			ah.set_succeeded() # full success
 		return ah
 
+	## Allows to trigger actions of the type actionlib/TestAction
+	#
+	# Based on the component and action name, the corresponding ActionServer will be called.
+	#
+	# \param component_name Name of the component.
+	# \param action_name Name of the action.
+	# \param blocking Whether to wait for the Action to return.
+	def trigger_action(self,component_name,action_name,blocking=True):
+		ah = action_handle(action_name, component_name, "", blocking, self.parse)
+		if(self.parse):
+			return ah
+		else:
+			ah.set_active(mode="action")
+
+		rospy.loginfo("<<%s>> <<%s>>", action_name, component_name)
+		#parameter_name = self.ns_global_prefix + "/" + component_name + "/service_ns"
+		#if not rospy.has_param(parameter_name):
+			#message = "Parameter " + parameter_name + " does not exist on ROS Parameter Server, aborting..."
+			#rospy.logerr(message)
+			#ah.set_failed(2, message)
+			#return ah
+		#service_ns_name = rospy.get_param(parameter_name)
+		action_full_name = "/" + component_name + "/" + action_name
```

Comment:
I kind of did not use the parameter approach on purpose as imho this mechanism is outdated and now only produces quite some (avoidable) configuration overhead. Maybe we can get rid of this parameter approach for good after all...but I'm happy to discuss this further